### DISCUSSION TITLE
Lost access - account created with Google SSO

### DISCUSSION THREAD

**Discussion by lbrito1** [deepgram/community#927] [General help] [answered]
I created an account via Google SSO. Later I decided to cancel my business' Google Workspace, but I forgot about all the SSOs I had.

I tried resetting my password, but all I get is this email:
>Hi there, We tried to reset your account's password, however our records show you've created your account with google Single Sign On. Please try the Log in with google option on the Console Login page. Happy building! The Deepgram Team https://deepgram.com/

I don't have a paid plan, so I guess all I can do is post here for help.

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jpvajda** [comment, marked as answer]
@lbrito1  Could you email us at `devrel@deepgram.com` from the email you used to register with? Just mention this Github Discussion in the email and we can try to get this sorted out.
